lib/modulesync/git.rb

```diff
@@ -51,7 +51,7 @@ def self.pull(git_base, name, branch, project_root, opts)
       # Repo needs to be cloned in the cwd
       if !Dir.exist?("#{project_root}/#{name}") || !Dir.exist?("#{project_root}/#{name}/.git")
         puts 'Cloning repository fresh'
-        remote = opts[:remote] || (git_base.start_with?('file://') ? "#{git_base}/#{name}" : "#{git_base}/#{name}.git")
+        remote = opts[:remote] || (( git_base.start_with?('file://') or git_base.include? 'git-codecommit') ? "#{git_base}/#{name}" : "#{git_base}/#{name}.git")
```

Comment:
Setting `remote` in the config resolves this issue.

Reply:
Resolving this comment, as mentioned elsewhere, setting `remote` doesn't resolve the issue.